packages/react/src/patterns/sections/CardSection/CardSection.js

```diff
@@ -98,8 +102,8 @@ const _renderCards = (cards, containerRef) => (
                 src: ArrowRight20,
               },
             }}
-            target={card.cta.target}
             type="link"
+            id={`region-${card.heading}`}
```

Comment:
wouldn't card heading potentially have spaces?

Reply:
@jeffchew - card heading has spaces I added hyphen for readable

Reply:
Having an id with spaces is generally not a good idea though.

Reply:
I added a uniqueid utility in this PR which might help: https://github.com/carbon-design-system/ibm-dotcom-library/pull/1547

Reply:
yeah @jeffchew but I need to use the  div which contains the card with aria-labelledby. and associate its id with the heading for that particular card.

Reply:
Right, what I'm suggesting is use the uniqueid utility to generate an id, and use that as the id. Not the card heading, since those can introduce any tyles of characters that would make the id name invalid (special characters, spaces, etc).

Reply:
ok. @jeffchew- I believe the utility is not merged?

Reply:
Yes, it needs to be reviewed and merged.